pkg/sensors/sensors.go

```diff
@@ -83,6 +85,12 @@ func SensorBuilder(name string, p []*program.Program, m []*program.Map) *Sensor
 	}
 }
 
+type policyHandler interface {
+	// PolicyHandler returns a Sensor for a given policy
+	PolicyHandler(policy tracingpolicy.TracingPolicy, filterID policyfilter.PolicyID) (*Sensor, error)
+}
+
+// NB: deprecated, please use policyHandler
```

Comment:
Is there a reason we have to deprecate this instead of just getting rid of it?

Reply:
Just to reduce the number of patches. Plan is to move them all. Made an issue here: https://github.com/cilium/tetragon/issues/785